test/features/caseFilters.feature

```diff
@@ -3,9 +3,72 @@ Feature: Set of scenarios to test functionality of create case filters in case c
+            | Text        | Text Field                                                         |                                                            |
+            | TextArea    | Text Area                                                          |                                                            |
+            | Date        | Date Field                                                         |                                                            |
+            | Complex     | Address Field,Address Line 1,Address Line 2,Address Line 3,Country |                                                            |
+            | Phone-UK    | Phone Field                                                        |                                                            |
+            | Number      | Number Field                                                       |                                                            |
+            | Yes-No      | Yes or No Field                                                    |                                                            |
+            | Collection  | Collection Field                                                   |                                                            |
+            | Fixed-List  | Marrital Status Field                                              | --Select a value--,Marriage,Civil Partnership,Single,Widow |
+            | Money-GBP   | Money Field                                                        |                                                            |
+            | Document    | Document Field                                                     |                                                            |
+            | Multi-Select| Multi Select Field,Cardiff,Manchester,Oxford                       |                                                            |
+            | Email       | Email Field                                                        |                                                            |
+
+
+    @searchtest @run
+    Scenario Outline: reset button clears drop down options and removes all dynamic filters
+        And I have filled out the search filters including dynamic filters
+        When I click the 'Reset' button
+        Then I should not see a '<dataType>' dynamic filter
+
+        Examples:
+            | dataType    |
+            | TextArea    |
+#            | Date        |
+#            | Complex     |
+#            | Phone-UK    |
+#            | Number      |
+#            | Yes-No      |
+#            | Collection  |s
+#            | Fixed-List  |
+#            | Money-GBP   |
+#            | Document    |
+#            | Multi-Select|
+#            | Email       |
+
+      # due to current functionality of system on aat when clicking reset button it switches to another default
```

Comment:
Do we need to update the definition file then? Not sure I understand why we are commenting all out?

Reply:
needs updating from me

Reply:
to uncomment